features/__init__.py

```diff
@@ -10,6 +12,10 @@ def import_features(config_dict):
             _FEATURES[feature_name] = globals()[feature_name] = FeatureNameValue(
                 feature_name, feature_val
             )
+    for key, val in os.environ.items():
+        if key.startswith("QUAY_FEATURE_"):
```

Comment:
In some cases we don't need FEATURE_SECURITY_SCANNER in our local-dev, so I added ability to enable/disable features via environment variables.